src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -235,7 +236,7 @@ CleanupTaskExecution(TaskExecution *taskExecution)
 	pfree(taskExecution->taskStatusArray);
 	pfree(taskExecution->connectionIdArray);
 	pfree(taskExecution->fileDescriptorArray);
-	memset(taskExecution, 0, sizeof(TaskExecution));
+	pfree(taskExecution);
```

Comment:
This function is called towards the end of ```MultiRealTimeExecute()``` and ```TrackerCleanupResources()```

There are no references left pointing here.